### DISCUSSION TITLE
Prima riunione degli ambassadors - Cosa, come e quando?

### DISCUSSION THREAD

**Discussion by Cadienvan** [Il-Libro-Open-Source/book#37] [Salotto]
I primi 15 ambassador (+ io) sono stati nominati (Ancora complimenti e benvenuti 🚀) e nasce ora la necessità di aprire un canale di comunicazione per discutere dello stato del progetto e di come procedere da qui in poi.

Pensavo di aprire un sondaggio ma per evitare che erroneamente qualche non-ambassador possa rispondere confondendo il risultato apro questa discussione e chiedo a tutti gli @Il-Libro-Open-Source/ambassadors di commentare con le loro preferenze, ma nell'ottica di apertura verso l'intera community è corretto che anche questa discussione rimanga aperta e accessibile.

Propongo alcune date nella speranza possano combaciare con la maggior parte dei vostri calendari, vi chiedo di commentare esprimendo una o più preferenze. La prima riunione durerà all'incirca 90 minuti.

1. Venerdì 8 Settembre, ore 18:00.
2. Venerdì 8 Settembre, ore 20:30.
3. Mercoledì 13 Settembre, ore 18:00.
4. Mercoledì 13 Settembre, ore 20:30.
5. Sabato 16 Settembre, ore 15:00.
6. Sabato 16 Settembre, ore 18:00.
7. Sabato 16 Settembre, ore 20:30.
8. Martedì 19 Settembre, ore 18:00.
9. Martedì 19 Settembre, ore 20:30.
10. Sabato 23 Settembre, ore 15:00.
11. Sabato 23 Settembre, ore 18:00.
12. Sabato 23 Settembre, ore 20:30.
13. Giovedì 21 Settembre, ore 18:00.
14. Giovedì 21 Settembre, ore 20:30.

Pensavo di inviare un link su Meet / Zoom, a riguardo avete preferenze?
Se possibile, il tutto verrà registrato e fornito a chi non potrà partecipare.

L'ordine del giorno sarà (Aggiungete nei commenti se mi sono perso qualcosa):
- Presentazioni (Proprio 30 secondi a testa, molti di noi si conoscono già ed è giusto un modo per introdursi al gruppo). Time slot: 10 minuti.
- Metodo di lavoro: Definizione del metodo con il quale si vuole portare avanti lo sviluppo del libro. Discussioni su branch / PR / formato del libro / suddivisione delle folder e dei file. Time slot: 30 minuti (+10 se necessario).
- Punti principali / Capitoli: Analizzare quanto è stato proposto nel thread #3 e capire cosa portare avanti, cosa tenere in stand-by e cosa bocciare. L'idea è quella di individuare i punti principali più che i capitoli. Questo resta chiaramente un primo momento a cui ne seguiranno altri via via che il progetto cresce. Time slot: 15 minuti.
- Licenza: Se qualcuno avesse competenze a riguardo, sarebbe utile scegliere la licenza del progetto. La discussione verrà rimandata in mancanza di expertise. Time slot: 5 minuti.
- Smistamento degli ambassadors: Una prima suddivisione delle aree di influenza e di interesse di ogni membro del team di modo che chi vuole possa cominciare già a buttare giù delle idee e degli appunti. Time slot: 15 minuti.
- Nomina ulteriori ambassadors: Capire assieme ogni quanto portare a termine una tornata di votazioni per portare in casa nuovi membri del team. Time slot: 5 minuti.
- Next step: Come riunirci le prossime volte e _prenotazione_ prossimo slot per fine settembre / inizio ottobre.

Come è ipotizzabile aspettarsi, è possibile che alcuni degli argomenti non verranno trattati a pieno, per questo l'ultimo punto è un *Next step* per prendersi già del tempo a seguire.

**Comment by serenasensini** [comment]
For me:

Mercoledì 13 Settembre, ore 18:00
Martedì 19 Settembre, ore 18:00
Giovedì 21 Settembre, ore 18:00

**Comment by corradopetrelli** [comment]
Per me:

- Venerdì 8 Settembre, ore 18:00
- ~Venerdì 8 Settembre, ore 20:30~
- Mercoledì 13 Settembre, ore 18:00
- Mercoledì 13 Settembre, ore 20:30

**Reply by corradopetrelli** [reply]
Scusami tanto @Cadienvan, non ricordavo di un impegno l'8 settembre a cena. Aggiorno il commento.

**Comment by imD3v** [comment]
Ciao Michael,
Per me : 

- Mercoledì 13 Settembre, ore 18:00.
- Martedì 19 Settembre, ore 18:00.
- Giovedì 21 Settembre, ore 18:00.

Ovviamente questi sono gli orari "comodi" ma se dovesse servire esserci in qualche altro giorno/orario ci si organizza e ci sono!

Meet o Zoom nessuna differenza per me, come nessuna problematica in merito alla registrazione della call.

Consiglierei di aggiungere in ODG la possibilità di aprire un canale più smart per queste comunicazioni/sondaggi cosa ne pensi?

**Reply by Cadienvan** [reply]
Top, d'accordo per il canale!

**Reply by corradopetrelli** [reply]
> Consiglierei di aggiungere in ODG la possibilità di aprire un canale più smart per queste comunicazioni/sondaggi cosa ne pensi?

Condivido pure io, penso che sarebbe comodo un gruppo telegram o un canale discord

**Reply by JeyDi** [reply]
Se posso dare un'opinione: telegram sarebbe meglio :P

**Reply by corradopetrelli** [reply]
> Se posso dare un'opinione: telegram sarebbe meglio :P

Anche io preferirei telegram dato che lo uso quotidianamente

**Reply by eppak** [reply]
Anche per me telegram va bene

**Comment by JeyDi** [comment]
Ciao! 
Concordo anche io di aggiungere nell'ODG la possibilità di avere un canale di comunicazione dedicato (che non siano le discussioni github).
Per quanto riguarda lo strumento meglio zoom in quanto consente di registrare.
La minuta dell'incontro la possiamo poi condividere qui in una discussione dedicata.

Disponibilità (prima è meglio è):
- Venerdì 8 settembre, ore 20:30
- Mercoledì 13 Settembre, ore 18:00.
- Mercoledì 13 Settembre, ore 20:30.
- Sabato 16 Settembre, ore 15:00.
- Sabato 23 Settembre, ore 15:00.
- Giovedì 21 Settembre, ore 18:00.
- Giovedì 21 Settembre, ore 20:30.

**Comment by eppak** [comment]
Salve, come canale, come scritto sopra va bene telegram come anche whatsapp; la riunione per me anche qui va bene un po' tutto da un meet su google a zoom. 
Per me vanno bene queste date:

    Venerdì 8 settembre, ore 20:30
    Mercoledì 13 Settembre, ore 20:30.
    Giovedì 21 Settembre, ore 20:30.

**Comment by francescobianco** [comment]
Ciao @Cadienvan e grazie per tenermi e tenerci attivamente coinvolti al progetto

ecco le mie preferenze:

* Venerdì 8 settembre, ore 20:30.
* Sabato 16 Settembre, ore 15:00.
* Martedì 19 Settembre, ore 18:00
* Sabato 23 Settembre, ore 15:00.
* Giovedì 21 Settembre, ore 18:00.
* Giovedì 21 Settembre, ore 20:30.

Anche per me Telegram non sarebbe male per le comunicazioni interne.

**Comment by passasooz** [comment]
Ciao @Cadienvan io temo che Venerdì 8 settembre non posso esserci, mentre per le altre date che hai proposto va bene!

Per me va bene telegram o whatsapp è uguale

**Comment by elgorditosalsero** [comment]
Salve @Cadienvan, purtroppo non posso dare disponibilità per nessuna data al momento, in quanto sarò a Dublino per lavoro e non so ancora quando torno (e 3/4 delle sere le ho già impegnate con eventi aziendali).

Se dovesse cambiare qualcosa, ti aggiorno!

Per le comunicazioni: telegram >>>>>>> all

**Reply by Cadienvan** [reply]
No problem, come dicevo cercheremo di registrare il tutto!

**Comment by nicolaerario** [comment]
Grazie dei complimenti, li giro a tutto il team.

I miei slots:

1. Venerdì 8 Settembre, ore 18:00.
2. Mercoledì 13 Settembre, ore 18:00.
3. Giovedì 21 Settembre, ore 18:00.

Preferirei Meet

🤙

**Comment by GuidoPenta** [comment]
Ciao! Per me telegram sarebbe top assoluto. 
Come disponibilità tendenzialmente quando vi pare tranne lunedì sera (preferisco di gran lunga in orario d'ufficio rispetto alla sera)

**Comment by fabiobiondi** [comment]
Ciao mi intrometto per un consiglio inutile (e non richiesto) : 
per proporre e scegliere le date di un meeting un tool comodissimo è Doodle

**Reply by Cadienvan** [reply]
Grazie Fabio, ecco come si chiamava!!
Ce lo teniamo nel cassetto, grazie!

**Comment by Cadienvan** [comment]
Ragazzi ho aperto un gruppo Telegram per ambassador (Al momento, poi estenderemo se vediamo che ne vale la pena), aggiungete Cadienvan così vi invito.

**Reply by Cadienvan** [reply]
@Il-Libro-Open-Source/ambassadors vi taggo così ricevete la notifica.

**Comment by stefanotorresi** [comment]
Io ho disponibilità un po' limitata questo mese.
> Mercoledì 13 Settembre, ore 18:00.

**Comment by davideimola** [comment]
Ciao a tutti!

Io purtroppo tra quei giorni posso solo Martedì 19!

**Comment by halfpass25** [comment]
Anche se la mia candidatura a traduttore è ancora sub iudice, per il canale direi che Zoom o Meet o comunque qualcosa che consente di registrare per chi non può esserci, mi sembra meglio di scorrere un milione di interazioni su WA o Telegram.
Magari a seguire, uno di questi due ultimi canali può essere utilmente attivato per gestire le cose ordinarie, a regime.
Ma per la fase iniziale, guardiamoci negli occhi, se possibile ;)

**Comment by Developpoo** [comment]
Per ora sono molto impegnato ma se mi dovessi proporre per un capitolo potrei farlo, se può interessare, per i risvolti fiscali che ci possono essere… avendo con mia moglie uno studio di consulenza dovrei avere facilità nel recepire informazioni utili in questo ambito… a prescindere da un progetto open source che quindi si presuppone a non scopo di lucro siamo in italia ed i risvolti fiscali ho imparato che ci sono SEMPRE!!!! Distinti Saluti

**Reply by Cadienvan** [reply]
@Developpoo ho modificato il tuo commento perché c'erano mail e numeri di telefono

**Comment by Cadienvan** [comment]
Lascio la discussione aperta per eventuali note di giornata ma confermo l'appuntamento mercoledì 13 ore 18:00 oltre che Telegram come strumento di comunicazione. Tutti gli ambassador meno 2 sono già presenti in gruppo.

**Comment by Cadienvan** [comment]
Aggiungo due note di cui credo valga la pena parlare:
1. Avendo messo in piedi la org possiamo meglio separare il libro nei suoi vari aspetti. Magari valutare l'idea di aprire le discussioni a livello di org e non di repo?
2. Il titolo del libro, potremmo cominciare a vagliare qualche idea.

**Reply by guizzo** [reply]
~2. _Developer's path_ è troppo scontato vero? Si hai ragione, è davvero scontato~

~EDIT: anche _Developer's creed_ (cit.) mi sà che è un po' cacofono...~

EDIT 2: riflettendoci sopra confermo quanto detto: non si possono sentire, torno nel mio antro 😅

**Comment by guizzo** [comment]
@Cadienvan Anche qui, penso si possa chiudere...
